src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2955,4 +2955,7 @@
   <data name="ShowHiddenItems" xml:space="preserve">
     <value>Show hidden items</value>
   </data>
+  <data name="FormatDriveText" xml:space="preserve">
+    <value>Format...</value>
```

Comment:
Why the "..."?

Reply:
That's the default string used by windows (I suppose it's there because it pops up a menu)